packages/api/src/main.ts

```diff
@@ -6,18 +6,50 @@ import * as express from 'express';
 import { openfeature } from '@openfeature/openfeature-js';
 import { fibonacci } from '@openfeature/fibonacci';
 import { query, validationResult } from 'express-validator';
+import { OpenTelemetryHook, LoggingHook } from '@openfeature/extra';
+import { buildMarkup } from './markup';
 
 const app = express();
 
-const oFeatClient = openfeature.getClient('api');
+openfeature.registerHooks(new OpenTelemetryHook('test'));
```

Comment:
Give the OTel hook a better name. It's supposed to uniquely identify the component which generated the instrumentation.